.travis.yml

```diff
@@ -5,7 +5,7 @@ cache:
     - "$HOME/.composer/cache"
 
 php:
-  - 7
+  - 7.2
```

Comment:
When using the `php-7.2` test in Travis CI build, the PHPUnit will use the latest stable version.

The PHPUnit namespace changes into `PHPUnit\Framework\TestCase`.

It should be fixed and the Travis CI build will be successful.